tests/tensor_search/test_multimodal_tensor_combination.py

```diff
@@ -0,0 +1,289 @@
+                        "test-text-two.": {
+                            "weight": 1,
+                        },
+                        "https://raw.githubusercontent.com/marqo-ai/marqo/mainline/examples/ImageSearchGuide/data/image2.jpg": {
+                            "weight": 0,
+                        },
+                    },)
+
+            # ensure we only call multimodal-combination twice
+            assert len(mock_multimodal_combination.call_args_list) == 2
+
+            return True
+
+        assert run()
+
+
+    def test_multimodal_field_content_dictionary_validation(self):
+        tensor_search.create_vector_index(
+                        index_name=self.index_name_1, config=self.config, index_settings={
+                        IndexSettingsField.index_defaults: {
+                            IndexSettingsField.model: "ViT-B/32",
+                            IndexSettingsField.treat_urls_and_pointers_as_images: True,
+                            IndexSettingsField.normalize_embeddings:False
+                        }
+                    })
+
+        # invalid weight string
+        res = tensor_search.add_documents(config=self.config, index_name=self.index_name_1, docs=[
+            {
+                "combo_text_image": {
+                    "A rider is riding a horse jumping over the barrier.": {
+                        "weight-void": 1,
+                    },
+                    "https://raw.githubusercontent.com/marqo-ai/marqo/mainline/examples/ImageSearchGuide/data/image1.jpg": {
+                        "weight": 0,
+                    },
+                },
+                "_id": "123",
+            },], auto_refresh=True)
+        assert res["errors"] == True
+
+        # invalid weight format string
+        res = tensor_search.add_documents(config=self.config, index_name=self.index_name_1, docs=[
+            {
+                "combo_text_image": {
+                    "A rider is riding a horse jumping over the barrier.": {
+                        "weight-void": "1.0",
+                    },
+                    "https://raw.githubusercontent.com/marqo-ai/marqo/mainline/examples/ImageSearchGuide/data/image1.jpg": {
+                        "weight": 0,
+                    },
+                },
+                "_id": "123",
+            }, ], auto_refresh=True)
+        assert res["errors"] == True
+
+        # invalid field content format
+        res = tensor_search.add_documents(config=self.config, index_name=self.index_name_1, docs=[
+            {
+                "combo_text_image": {
+                    "A rider is riding a horse jumping over the barrier.": 0.5,
+                    "https://raw.githubusercontent.com/marqo-ai/marqo/mainline/examples/ImageSearchGuide/data/image1.jpg": {
+                        "weight": 0,
+                    },
+                },
+                "_id": "123",
+            }, ], auto_refresh=True)
+        assert res["errors"] == True
```

Comment:
Can you assert that you can't find it in the index?

Reply:
done